### PR TITLE
Include pre-commit plugin 

### PR DESCRIPTION
**Pull request by awebdeveloper**
Included info to use with yelp's pre-commit

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -157,7 +157,26 @@ In the future we will have better support for formatting whole projects.
 
 #### Pre-commit hook for changed files
 
-[lint-staged](https://github.com/okonet/lint-staged) can re-format your files that are marked as "staged" via `git add`  before you commit.
+You can use this with a pre-commit tool. This can re-format your files that are marked as "staged" via `git add`  before you commit.  
+
+##### [pre-commit](https://github.com/pre-commit/pre-commit)
+
```
Comment (justingreenberg) [inline README.md]: typo: comfig -> config

Comment (awebdeveloper) [inline README.md]: fixed

Comment (awebdeveloper) [inline README.md]: @justingreenberg any other changes you need

**Comment by vjeux** [comment]
Awesome, thanks for building that! Do you mind putting yours after lint-staged? It seems like lint-staged is the most popular one with prettier right now and has been battle tested, so I'd like it first. Once you do that, I'll merge it in!

**Comment by awebdeveloper** [comment]
@vjeux  done

**Comment by vjeux** [comment]
Thanks!!

**Pull request merged**